Improve doc for logLevel option (#542)
DataDog/serverless-plugin-datadog: README.md

| `customHandler`               | When set, the specified handler is set as the handler for all the functions.                                                                                                                                                                                                                                                                                                                                                 |
| `failOnError`                 | When set, this plugin throws an error if any custom Datadog monitors fail to create or update. This occurs after deploy, but will cause the result of `serverless deploy` to return a nonzero exit code (to fail user CI). Defaults to `false`.                                                                                                                                                                              |

> | `logLevel`                    | The log level for Datadog's lambda extension and tracing libraries. Set to `DEBUG` for extended logging. It does not affect which user logs will be forwarded to Datadog.                                                                                                                                                                                                                                                                                                                                                                         |

Reviewer: ```suggestion
| `logLevel`                    | The log level for Datadog's lambda extension and tracing libraries. Set to `DEBUG` for extended logging. It does not affect which user logs are forwarded to Datadog.                                                                                                                                                                                                                                                                                                                                                                         |
```
